src/components/flow/flow.e2e.ts

```diff
@@ -21,214 +22,245 @@ describe("calcite-flow", () => {
     expect(element).not.toHaveClass(CSS.frameRetreating);
   });
 
-  it("back() method should remove item", async () => {
-    const page = await newE2EPage();
+  describe("works with flow-items and panels (deprecated)", () => {
+    testItemBehavior("flow-item");
 
-    await page.setContent("<calcite-flow><calcite-panel></calcite-panel></calcite-flow>");
+    testItemBehavior("panel");
 
-    const flow = await page.find("calcite-flow");
+    function testItemBehavior(itemType: "flow-item" | "panel"): void {
+      const itemTag = `calcite-${itemType}`;
 
-    await flow.callMethod("back");
+      it("back() method should remove item", async () => {
+        const page = await newE2EPage();
 
-    await page.waitForChanges();
+        await page.setContent(`<calcite-flow><${itemTag}></${itemTag}></calcite-flow>`);
 
-    const flowItem = await page.find("calcite-panel");
+        const flow = await page.find("calcite-flow");
 
-    expect(flowItem).toBeNull();
-  });
+        await flow.callMethod("back");
 
-  it("setting 'beforeBack' should be called in 'back()'", async () => {
-    const page = await newE2EPage();
+        await page.waitForChanges();
 
-    const mockCallBack = jest.fn().mockReturnValue(Promise.resolve());
-    await page.exposeFunction("beforeBack", mockCallBack);
+        const flowItem = await page.find(itemTag);
 
-    await page.setContent("<calcite-flow><calcite-panel></calcite-panel></calcite-flow>");
+        expect(flowItem).toBeNull();
+      });
 
-    await page.$eval(
-      "calcite-panel",
-      (elm: HTMLCalcitePanelElement) =>
-        (elm.beforeBack = (window as typeof window & Pick<typeof elm, "beforeBack">).beforeBack)
-    );
+      it("goes back when item back button is clicked", async () => {
+        const page = await newE2EPage();
 
-    const flow = await page.find("calcite-flow");
+        await page.setContent(`<calcite-flow><${itemTag}></${itemTag}></calcite-flow>`);
```

Comment:
I think this needs 2 itemTag types for back button to show up.

Reply:
Good catch. Fixed.